package.json

```diff
@@ -147,8 +147,8 @@
     "@spectrum-css/component-builder": "^1.0.0"
   },
   "resolutions": {
-    "@babel/core": "7.8.4",
-    "@babel/runtime": "7.6.3",
+    "@babel/core": "7.8.0",
```

Comment:
this looks like its going `7.8.0` but the comment looks like it was supposed to go to `7.9.0`?

Reply:
yeah my upgrade was being a bit funny. I might update all the package.json entries to force it.